crates/solidity/testing/snapshots/cst_output/ContractDefinition/unterminated_body/generated/0.4.22.yml

```diff
@@ -5,7 +5,7 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected Identifier, or MappingKeyword, or FunctionKeyword, or UnsignedFixedType, or SignedFixedType, or UnsignedIntegerType, or SignedIntegerType, or FixedBytesType, or PayableKeyword, or AddressKeyword, or StringKeyword, or BoolKeyword, or ErrorKeyword, or EventKeyword, or TypeKeyword, or EnumKeyword, or StructKeyword, or ModifierKeyword, or FunctionKeyword, or FunctionKeyword, or ConstructorKeyword, or UsingKeyword, or CloseBrace
+    Error: Expected Identifier, or UnsignedFixedType, or SignedFixedType, or UnsignedIntegerType, or SignedIntegerType, or FixedBytesType, or PayableKeyword, or AddressKeyword, or StringKeyword, or BoolKeyword, or MappingKeyword, or FunctionKeyword, or ErrorKeyword, or EventKeyword, or TypeKeyword, or EnumKeyword, or StructKeyword, or ModifierKeyword, or FunctionKeyword, or FunctionKeyword, or ConstructorKeyword, or UsingKeyword, or CloseBrace
```

Comment:
Need to make sure this output is deterministic. Both before/after have the same names, but with different order. Also, `FunctionKeyword` is duplicated a few times.